lib/active_ldap/adapter/jndi_connection.rb

```diff
@@ -134,10 +134,8 @@ def search(options)
         else
           @context.set_request_controls([])
         end
-
-        escaped_base = escape_dn(base)
         loop do
-          @context.search(escaped_base, filter, controls).each do |search_result|
+          @context.search(escape_dn(base), filter, controls).each do |search_result|
```

Comment:
Could you revert this change?
This may cause multiple `escape_dn` calls.

Reply:
Updated